src/target/riscv/riscv.c

```diff
@@ -139,6 +139,35 @@ struct tdata1_cache {
 	struct list_head elem_tdata1;
 };
 
+bool riscv_target_config_allows_hw_translation(const struct target *target)
```

Comment:
Please, consider shortening the name of the function.

You can for example call it `riscv_is_virt2phys_hw_mode()` which is shorter, readable and still very clear.

```suggestion
bool riscv_is_virt2phys_hw_mode(const struct target *target)
```

Reply:
Addressed